File: Latihan B/java/Ram.java
// deklarasi kelas ram
public class Ram{
    // variabel privat
    private String capacity;
    private int price;
    // method public dan konstruktor
    public Ram(){

    }
    public Ram(String capacity, int price){
        this.capacity=capacity;
        this.price=price;
    }
    // get dans set semua variabel
    public String getcapacity(){
        return capacity;
    }
    public void setcapacity(String capacity){
        this.capacity = capacity;
    }
    public int getprice(){
        return price;
    }
    public void setprice(int price){
        this.price = price;
    }
}

File: Latihan B/python/Ram.py
# deklarasi kelas
class Ram:
    # deklarasi variabel
    __capacity = ""
    __price = 0
    # method dan konstruktor
    def __init__(self, capacity = "", price=0):
        self.__capacity = capacity
        self.__price = price
    #  get dans set semua variabel
    def setprice(self, price):
        self.__price = price
    def getprice(self):
        return self.__price
    def setcapacity(self, capacity):
        self.__capacity = capacity
    def getcapacity(self):
        return self.__capacity